Enterprise-CMCS/macpro-mdct-mfp
# WP - Initiatives Details - Close out

### Description



### Related ticket(s)


MDCT-2817

### What this PR covers
- close out UI 
- a new page type, which covers I. Define Initiative and IV. Close out
- opening and closing a modal
- renamed EntityDetailsOverlay to EntityDetailsDashboardOverlay (which @cassandradanger will be using/updating in future PR)

### What this PR does not cover
- the buttons don't actually work / close out the initiative 
- the sidebar is still visible because this overlay is yet to be nested under a dashboard 
- modal won't actually close out initiative, save & return is not hooked up yet
- Projected end date will not populate with a date yet
---
### How to test


- Login as a state user and start/edit an MFP work plan
- In the Sidebar, click the "State & territory specific initiatives" dropdown and click **IV. Close out** in the sidebar
- Confirm that the UI matches requirements and that the modal can open and close
- Head to I. Define Initiative in the sidebar and make sure the UI is still working as expected
- Ensure that changes meet feedback provided [here](https://qmacbis.atlassian.net/browse/MDCT-2817?focusedCommentId=123836)

https://github.com/Enterprise-CMCS/macpro-mdct-mfp/assets/6752256/4ad85f44-611a-4369-ac4c-7a13a95f3354

### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
